clients/privacy-center/pages/index.tsx

```diff
@@ -110,6 +110,7 @@ const Home: NextPage = () => {
         >
           <Image
             src={config.logo_path}
+            id="privacyCenterLogo"
```

Comment:
Please update the id value to `privacy-center-logo`. Best practice naming convention for CSS selectors.